src/e2ee/sharedKeyManagement.ts

```diff
@@ -15,10 +15,11 @@ limitations under the License.
 */
 
 import { useEffect, useMemo } from "react";
+import { Room } from "matrix-js-sdk";
```

Comment:
This is another top-level import (honestly, maybe it's not a problem?)

Reply:
mmm, better to be consistent probably